app/docs/0.10.x/plugin-development/custom-logic.md

```diff
@@ -30,7 +30,8 @@ Kong allows you to write your code in all of the lua-nginx-module contexts. Eac
 |-------------------------|------------------------------------|--------------
 | `:init_worker()`         | [init_worker_by_lua]               | Executed upon every Nginx worker process's startup.
 | `:certificate()`         | [ssl_certificate_by_lua_block]     | Executed during the SSL certificate serving phase of the SSL handshake.
-| `:access()`              | [access_by_lua]                    | Executed for every request upon it's reception from a client and before it is being proxied to the upstream service.
+| `:rewrite()`             | [rewrite_by_lua_block]             | Executed for every request upon it's reception from a client as a rewrite phase handler and before Kong matches the request to an API or a Consumer (and before the [access_by_lua] phase). *NOTE* in this phase neither the `api` nor the `consumer` have been identified, hence this handler will only be executed if the plugin was configured as a global plugin!
```

Comment:
typo: "it's" should be its , it appears it was carried over from the access description copy-paste